src/lib/components/Table/utils.ts

```diff
@@ -0,0 +1,34 @@
+// Table Utils
+
+// Cell Formatters ---
+
+/** Wrap object key value with HTML tags.  */
+export function tableCellFormatter(object: any[], key: string, openTag: string, closeTag: string): any[] {
```

Comment:
I've modified as follows:

- Now access `tagName` and `classes` as params
- Create a new entity using `createElement(tagName)`
- Appends the key value as the innerHTML
- Uses `setAttribute(classes)` to append optional classes
- Updates the object value using `outerHTML`

I *think* this will be better?

Reply:
Actually on second thought, this is not a good change to make. Svelte can't do anything with `document` until after the Mount event. This means all the table setup would have to occur during `onMount` which is slow as molasses.

This was just a utility feature, so I think for now I'll just disable it until we can think this through a bit more.

Reply:
Good point about the `onMount`, hadn't considered that.